config/locales/es.yml

```diff
@@ -512,6 +512,7 @@ es:
     search: Búsqueda
     security_html: Su información está segura y la información que proporcione será tratada de acuerdo con nuestro %{privacy_link}.
     select_file: Seleccionar archivo
+    select_prompt: "-Seleccionar-"
```

Comment:
maybe i should have put this under the state_file namespace to avoid i18n-tasks from complaining instead? but i think the correct translation is pretty clear?